build/main.go

```diff
@@ -109,24 +109,7 @@ func deps() {
 	log.Println("Installing dependencies...")
 
 	cmds := []command{
-		cmd("go get -u github.com/whyrusleeping/gx"),
-		cmd("go get -u github.com/whyrusleeping/gx-go"),
-		cmd("gx install"),
-		cmd("gx-go rewrite"),
-		cmd("go get -u github.com/golangci/golangci-lint/cmd/golangci-lint"),
-		cmd("go get -u github.com/stretchr/testify"),
-		cmd("go get -u github.com/xeipuuv/gojsonschema"),
-		cmd("go get -u github.com/ipfs/iptb"),
-		cmd("go get -u github.com/docker/docker/api/types"),
-		cmd("go get -u github.com/docker/docker/api/types/container"),
-		cmd("go get -u github.com/docker/docker/client"),
-		cmd("go get -u github.com/docker/docker/pkg/stdcopy"),
-		cmd("go get -u github.com/ipsn/go-secp256k1"),
-		cmd("go get -u github.com/json-iterator/go"),
-		cmd("go get -u github.com/prometheus/client_golang/prometheus"),
-		cmd("go get -u github.com/prometheus/client_golang/prometheus/promhttp"),
-		cmd("go get -u github.com/jstemmer/go-junit-report"),
-		cmd("go get -u github.com/pmezard/go-difflib/difflib"),
+		cmd("go mod download"),
```

Comment:
IMO there is no reason to run the download. It will download also dependencies that are not needed (dependencies of tests of packages we depend on or for other systems).

`go build ` and `go test` will download all modules needed.

Reply:
We don't _need_ to download but:

1. This helps us cache in CI. That is, we can download, cache, then build instead of building then caching.
2. Ideally, this should allow someone to fetch the deps and then work offline.